src/mainboard/clevo/adl-p/vboot-rwa.fmd

```diff
@@ -6,12 +6,12 @@ FLASH 32M {
 
 	UNUSED 11M
 
-	RW_MISC 424K {
+	RW_MISC 680K {
 		UNIFIED_MRC_CACHE(PRESERVE) 128K {
 			RECOVERY_MRC_CACHE 64K
 			RW_MRC_CACHE 64K
 		}
-		SMMSTORE(PRESERVE) 256K
+		SMMSTORE(PRESERVE) 512K
```

Comment:
1. Is it going to be required for other devices as well?
2. Why it has become a problem now? Anything changed, or simply not spotted before?
3. How much free space we have now after this change?